tmt/steps/provision/__init__.py

```diff
@@ -354,12 +354,10 @@ def details(self):
         kernel = self.execute('uname -r')[0].strip()
         self.verbose('kernel', kernel, 'green')
 
-    def _ansible_verbosity(self):
-        """ Prepare verbose level based on the --debug option count """
-        if self.opt('debug') < 3:
-            return ''
-        else:
-            return ' -' + (self.opt('debug') - 2) * 'v'
```

Comment:
Why have you disabled the original verbosity detection? It works fine with the corresponding debug levels and is useful for default inspection.